include/aspect/postprocess/visualization/geoid.h

```diff
@@ -0,0 +1,87 @@
+/*
+  Copyright (C) 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__postprocess_visualization_geoid_h
+#define __aspect__postprocess_visualization_geoid_h
+
+#include <aspect/postprocess/visualization.h>
+#include <aspect/simulator_access.h>
+#include <aspect/postprocess/geoid.h>
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      /**
+       * A class derived from CellDataVectorCreator that takes an output
+       * vector and computes a variable that represents the geoid
+       * topography. This quantity, strictly speaking, only makes sense at the
+       * surface of the domain. Thus, the value is set to zero in all the
```

Comment:
depending on the definition of "surface" you might add: "... and the bottom of the domain".